### PR TITLE
Underscore 1.8.3

### PR DESCRIPTION
**Pull request by bruennijs**
I ONLY added the underscore.create(...) func. I am new to the open source world an dI want to try. I couldn't push a feature branch in your repository (permission denied for me). So I created a fork on my remote.

Thanks, Olli

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "DefinitelyTyped",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (vvakame) [inline package.json]: do not edit package.json.

**Comment by vvakame** [comment]
NOTE: http://underscorejs.org/#create

**Comment by mhegazy** [comment]
Thank you for your contribution. 

Can you please fix the failing Travis CI tests and update the PR!

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**